src/main/features/dashboard/views/macro/claimStatus/partAdmitRejectionClaimant.njk

```diff
@@ -2,23 +2,38 @@
 {% from "internalLink.njk" import internalLink %}
 
 {% macro partAdmitRejectedByClaimantDashboard() %}
-  {{ t('You’ve rejected the defendant’s admission.') }}
+  {% if featureToggles.mediation and claim.claimantResponse.freeMediation === FreeMediationOption.YES %}
+    {{ t('We’ll contact you with a mediation appointment') }}
+  {% else %}
+    {{ t('You’ve rejected the defendant’s admission.') }}
+  {% endif %}
 {% endmacro %}
 
 {% macro partAdmitRejectedByClaimantClaimDetails(claim) %}
-  <div class="status-block">
-    <h2 class="heading-small">{{ t('We’ll contact you with a mediation appointment') }}</h2>
+  {% if featureToggles.mediation and claim.claimantResponse.freeMediation === FreeMediationOption.YES %}
+    <div class="status-block">
+      <h2 class="heading-small">{{ t('We’ll contact you to try to arrange a mediation appointment') }}</h2>
 
-    <p>{{ t('You rejected the defendant’s admission of {{ amount }}', { amount: claim.response.amount | numeral }) }}</p>
-
-    {% if claim.claimantResponse.freeMediation === FreeMediationOption.YES %}
-      <p>{{ t('You’ve both agreed to try mediation. We’ll contact you to arrange a call with the mediator.') }}</p>
+      <p>{{ t('You’ve rejected the defendant’s response.')}}</p>
+      <p>{{ t('You’ve both agreed to try mediation. We’ll contact you to try to arrange a call with the mediator.') }}</p>
       <p>{{ internalLink('Find out how mediation works', DashboardPaths.howFreeMediationWorksPage.uri) }}</p>
```

Comment:
Just noticing that it is missing from the original code too. can you please update it as well?